src/04-arrays.js

```diff
@@ -92,7 +105,9 @@ function addElementToEndOfArray(array, element) {}
  *  removeElementFromEndOfArray([ 10, 9, 8 ]);
  *  //> 8
  */
-function removeElementFromEndOfArray(array) {}
+function removeElementFromEndOfArray(array) {
+return array.pop(array.length-1);
```

Comment:
the `pop` method does not take an argument. The proper syntax would be `array.pop()`